zerver/lib/upload.py

```diff
@@ -90,7 +90,7 @@ def resize_avatar(image_data, size=DEFAULT_AVATAR_SIZE):
         im = Image.open(io.BytesIO(image_data))
         im = ImageOps.fit(im, (size, size), Image.ANTIALIAS)
     except IOError:
-        raise BadImageError("Could not decode avatar image; did you upload an image file?")
+        raise BadImageError("Could not decode image; did you upload an image file?")
```

Comment:
This change to existing logic should have been its own commit for clarity.  Merged this as c5ed119f89b1c6ad2caa31f2cdc89fa5cc2c285a.